get-telemetry.py

```diff
@@ -0,0 +1,101 @@
+import os
+import csv
+import requests
+import urllib
+from datetime import datetime, timedelta
+
+MAX_REQUEST_SIZE_HOURS = 48
+SECONDS_IN_HOUR = 3600.0
+VOLTUS_API_URL = os.getenv("VOLTUS_API_URL", "https://api.voltus.co")
+VOLTUS_API_KEY = os.getenv("VOLTUS_API_KEY", "secret")
+SITE_ID = os.getenv("SITE_ID")
+
+s = requests.Session()
+s.headers.update({"X-Voltus-API-Key": VOLTUS_API_KEY, "Accept": "application/json"})
+
+
+def compute_current_end_time(current_start_time, end_time):
```

Comment:
I would just inline this logic as it's only used in one place and I think it's easier to follow if you see it in context.